statsmodels/discrete/discrete_margins.py

```diff
@@ -345,11 +345,13 @@ def margeff_cov_with_se(model, params, exog, cov_params, at, derivative,
 def margeff():
     pass
 
+
 def _check_at_is_all(method):
     if method['at'] == 'all':
-        raise NotImplementedError("Only margeff are available when `at` is "
-                    "all. Please input specific points if you would like to "
-                    "do inference.")
+        raise ValueError("Only margeff are available when `at` is "
```

Comment:
I'm not sure whether NotImplementedError would be more appropriate.
i.e. some option combination might/should be added.

Reply:
My impression is that we want to signal to the user "An incorrect value was passed" (ValueError) and "Other values may be implemented in the future" (NotImplementedError).  My inclination is to raise ValueError and have the attached message supply the latter piece of info.  Thoughts?